torch/_dynamo/trace_rules.py

```diff
@@ -2182,6 +2182,7 @@
         "torch.xlogy",
         "torch.zero_",
         "torch.zeros",
+        "torch.zeros_like",
```

Comment:
`torch.zeros_like` was exposed as `torch.obj`, so it has no trace rule here. lol